### ISSUE TITLE
Enhancement - reverse order of cards in draft pack

### ISSUE THREAD

**Issue by tiera3** [magefree/mage#11565]
Currently, draft pack cards are ordered from rarity (lowest to highest) starting with the land - which means if a player has a problem that results in an autopick, it selects the lowest rarity card for the autopick. If the order was reversed, then it would select one of the highest rarity cards.

(Also this change would alleviate the problem I am having where in Window mode, the oracle text doesn't popup for the bottom row of draft cards. Jay-Di was unable to replicate that problem however.) (By alleviate - I mean that I would get oracle text for the higher rarity half of the pack rather than the lower rarity half.)

**Comment by xenohedron** [comment]
Of course, a better solution would be for the autopick to use the draftbot logic.
